goclean.sh

```diff
@@ -8,7 +8,7 @@
 set -ex
 
 # Automatic checks
-test -z "$(gometalinter --disable-all --enable=gofmt $(glide novendor) | tee /dev/stderr)"
+test -z "$(gometalinter --disable-all --enable=gofmt --enable gosimple --deadline=4m $(glide novendor) | tee /dev/stderr)"
```

Comment:
if it is that slow i'm not so sure we actually want it running every time on the CI infrastructure.